catalog-info.yaml

```diff
@@ -9,5 +9,5 @@ metadata:
   #   backstage.io/techdocs-ref: dir:.
 spec:
   type: website
-  owner: group:bcgov/exchange-lab-developer-portal-team
+  owner: group:exchange-lab-developer-portal-team
```

Comment:
Removed the namespace based on [this documentation ](https://backstage.io/docs/features/software-catalog/references/#string-references)